libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -287,24 +377,13 @@ void HarmonicNotchFilter<T>::update(uint8_t num_centers, const float center_freq
         }
 
         const float notch_center = constrain_float(center_freq_hz[center_n] * (harmonic_n+1), 0.0f, nyquist_limit);
+        const uint8_t harmonic_mul = (harmonic_n+1);
         if (_composite_notches != 2) {
-            // only enable the filter if its center frequency is below the nyquist frequency
-            if (notch_center < nyquist_limit) {
-                _filters[_num_enabled_filters++].init_with_A_and_Q(_sample_freq_hz, notch_center, _A, _Q);
-            }
+            set_center_frequency(_num_enabled_filters++, notch_center, 1.0, harmonic_mul);
```

Comment:
Ditto comment above about _num_enabled_filters advancing even if the notch is above the nyquist

Reply:
see above comment